wpilibj/src/main/java/edu/wpi/first/wpilibj/apriltag/AprilTagFieldLayout.java

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj.apriltag;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Past experience is that the jackson imports can take a long time on the Rio.  Has anyone benchmarked this to make sure we aren't surprised?

Reply:
If they do take a long time, should we start shopping for a better JSON library, or write our own that wraps the C++ one?

Reply:
Unlike PathWeaver JSONs that would typically be loaded at the start of autonomous, this field layout JSON can be loaded once on robot startup. The loading time shouldn't be an issue.

Reply:
I benchmarked on Rio and while Pose3d has a significant first use time of ~150 ms, this is not due to Jackson.